repos/fdbt-site/src/data/auroradb.ts

```diff
@@ -847,9 +849,62 @@ export const getPassengerTypesByNocCode = async <T extends keyof SavedPassengerT
                 (row) =>
                     ({ id: row.id, name: row.name, passengerType: JSON.parse(row.contents) } as SavedPassengerType[T]),
             );
+        } else {
+            // filter out the ones with IDs that are created in global settings
+            return queryResults
+                .map((row) => JSON.parse(row.contents) as GroupPassengerType | GroupPassengerTypeDb)
+                .filter((row) => !row.companions.some((it) => 'id' in it)) as SavedPassengerType[T][];
```

Comment:
Is it possible to find a better name than `it`?

```
{"name":"Friends","maxGroupSize":"3","companions":[{"id":1,"minNumber":"2","maxNumber":"3"}]} 
```

Reply:
Ah yeah that's my Kotlin days catching up with me 😄